### PR TITLE
util/: Fix rev.go and compress.go

### PR DESCRIPTION
**Pull request by kcalvinalvin**
Fixed a bug in RevOffset generation and another in decompressing the
PKscript.

### CODE REVIEW

File: cmd/util/rev.go [hunk 1]
```diff
@@ -229,16 +229,14 @@ func writeOffset(fileNum uint32, offsetFile *os.File) error {
 	offset := uint32(0)
 	loc := int64(0)
 	// Read until the location of the offset is bigger than that of the file size
```
Comment (ysangkok) [inline cmd/util/rev.go:231]: now this comment doesn't match the code

File: cmd/util/rev_test.go [hunk 2]
```diff
@@ -47,7 +55,12 @@ func TestGetOneRevBlock(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	rb, err := GetRevBlock(382, RevOffsetFilePath) // fetches block 383
+	rb, err := GetRevBlock(299995, RevOffsetFilePath) // fetches block 383
```
Comment (ysangkok) [inline cmd/util/rev_test.go:58]: this comment is now incorrect

File: cmd/util/rev.go (hunk 1 above)
Comment (kcalvinalvin) [reply cmd/util/rev.go]: Whoops. Pushed a fix

File: cmd/util/rev_test.go (hunk 2 above)
Comment (kcalvinalvin) [reply cmd/util/rev_test.go]: Also pushed a fix

**Comment by adiabat** [comment]
awesome thanks!  Now for getting the real leaves in and out (#108)

**Pull request merged**